proto/penumbra/penumbra/core/component/dex/v1/dex.proto

```diff
@@ -632,6 +632,22 @@ message EventSwap {
   crypto.tct.v1.StateCommitment swap_commitment = 4;
 }
 
+// Tracks the LQT eligible volume for a position.
+// The volume corresponds to the outflow of staking tokens from
+// the position.
+message EventLqtPositionVolume {
```

Comment:
Is there a good way to link this to the execution of the dex? Like it could be nice to put in a bsod reference or something which would let us know whether or not it ties in to block execution or arb.

Reply:
That's a good idea, agreed. I think we will have to switch things around. We can do this on a second pass. One way to do this is to emit the event on `position_execution`. It will take some workshopping to do this without disrupting the method.